torch/csrc/api/include/torch/serialize.h

```diff
@@ -72,4 +102,38 @@ void load(Value& value, LoadFromArgs&&... args) {
   archive.load_from(std::forward<LoadFromArgs>(args)...);
   archive >> value;
 }
+
+/// Deserializes the given `tensor_vec` of type `std::vector<torch::Tensor>`.
+///
+/// To perform the serialization, a `serialize::InputArchive` is constructed,
+/// and all arguments after the `value` are forwarded to its `load_from` method.
+/// For example, you can pass a filename, or an `istream`.
+///
+/// \rst
+/// .. code-block:: cpp
+///
+///   std::vector<torch::Tensor> tensor_vec;
+///   torch::load(tensor_vec, "my_tensor_vec.pt");
+///
+///   std::vector<torch::Tensor> tensor_vec;
+///   std::istringstream stream("...");
+///   torch::load(tensor_vec, stream);
+/// \endrst
+template <typename... LoadFromArgs>
+void load(std::vector<torch::Tensor>& tensor_vec, LoadFromArgs&&... args) {
```

Comment:
can we not make this generic?  Consider the optimizer serialization below, which doesn't special case either the collection type nor the element type.